ngraph/core/src/op/grn.cpp

```diff
@@ -22,29 +17,25 @@ using namespace ngraph;
 
 NGRAPH_SUPPRESS_DEPRECATED_START
```

Comment:
we can remove `NGRAPH_SUPPRESS_DEPRECATED_START` as well

Reply:
Of course, done. Thanks!